llvm/include/llvm/Object/ELFObjectFile.h

```diff
@@ -1349,6 +1349,13 @@ template <class ELFT> Triple::ArchType ELFObjectFile<ELFT>::getArch() const {
     return Triple::UnknownArch;
   }
 
+  case ELF::EM_CUDA: {
+    if (EF.getHeader().e_ident[ELF::EI_CLASS] == ELF::ELFCLASS32)
+      return Triple::nvptx;
+    else
+      return Triple::nvptx64;
```

Comment:
avoid else-after-return, per the LLVM style guide. 
Instead use:
```
if (...)
  return ...;
return ...;
```

Reply:
There's a test for this now in https://github.com/llvm/llvm-project/pull/76992. I can do the style NFC if needed.

Reply:
Generally the test should go with the patch that introduces the change in behavior. If the change in behavior is unreachable/untestable without some other changes, then they should go together so test coverage is clear (otherwise it's hard to track that the patch did eventually get tested).

Sure - an NFC change would be good - no need to send it for review, you can commit that directly.

Reply:
There weren't any existing tests for any of this code, so I just punted it until the second patch. Thanks, I'll do the NFC.